mmv1/templates/terraform/examples/target_https_proxy_certificate_manager_certificate.tf.erb

```diff
@@ -0,0 +1,46 @@
+
+resource "google_compute_target_https_proxy" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]['target_https_proxy_name'] %>"
```

Comment:
Nit: Spacing is a little inconsistent in this file (like the space between "name" and "="). Possibly worth fixing since this will show up in the documentation website.